prombench/docs/kind.md

```diff
@@ -114,7 +114,20 @@ If used with the GitHub integration:
    export PR_NUMBER=<PR to benchmark against the selected $RELEASE>
    ```
 
-2. Deploy the Kubernetes objects:
+2. Before applying benchmarking objects , You have two choices to make:  
+- **Option 1: Download data from object storage**
```

Comment:
How would someone coming to this for the first time understand this choice?
Can we give some context why data might be downloaded?